python/llm/src/ipex_llm/transformers/models/llama.py

```diff
@@ -186,7 +186,14 @@ def llama_mlp_forward(
             hidden_states = attn_output.view(x.shape)
         return hidden_states
     else:
-        out = self.down_proj(self.act_fn(self.gate_proj(x)) * self.up_proj(x))
+        if os.environ.get("IPEX_LLM_LOW_MEM", None) == "1":
```

Comment:
This may not need to environment control?
Splitting the implementation won't impact the performance?

Reply:
Sure, test bigdl_fp16_gpu API on ARC, results:
before split:
0,meta-llama/Llama-2-7b-chat-hf,492.33,36.25,0.0,1024-128,1,1025-128,1,sym_int4,N/A,30.71,13.837890625,N/A
after split:
0,meta-llama/Llama-2-7b-chat-hf,493.1,35.49,0.0,1024-128,1,1025-128,1,sym_int4,N/A,7.57,13.6640625,N/A
This split should not influence the performance, will remove the environment variable check.